examples/src/basic/endpoint.rs

```diff
@@ -1,22 +1,22 @@
+use http::{Request, Response};
 use kinetics::macros::endpoint;
-use lambda_http::{Body, Error, Request, Response};
 use serde_json::json;
 use std::collections::HashMap;
+use tower::BoxError;
 
 /// REST API endpoint which responds with JSON {"success": true}
 ///
 /// Test locally with the following command:
 /// kinetics invoke BasicEndpointEndpoint
 #[endpoint(url_path = "/endpoint")]
 pub async fn endpoint(
-    _event: Request,
+    _event: Request<String>,
```

Comment:
@quesabe is there a way to avoid changes to the functions signatures?

Reply:
`http::Request` is generic and requires a parameter for body type. Thus the only way is to create our own request struct. `lambda_http:Request` is and alias for `http:Request<lambda_http::Body>` - that's why it's not generic and we can go the same path. Though effectively it does not change the signature, but only hides the body type under the hood.

Reply:
Ok, let's keep it like it currently is, in order to not introduce new abstractions.